src/components/app/App.js

```diff
@@ -1,5 +1,19 @@
 import React from 'react';
+import { 
+  BrowserRouter as Router,
+  Route, 
+  Switch
+} from 'react-router-dom';
+import { TopWords, getFirstWords } from '../../containers/words/TopWords';
```

Comment:
```suggestion
import TopWords from '../../containers/words/TopWords';
```
Right now TopWords is the default export coming from this file, it looks like getFirstWords is a selector.